medical_compliance/medical_compliance/api/tasks.py

```diff
@@ -74,6 +74,6 @@ def fetch_heart_rate_measurement():
         )
         heart_rate_measurement.save()
 
-    analyze_heart_rates.delay()
+    analyze_heart_rates.delay(last_meas)
```

Comment:
That's fair enough, I don't even know why I used that naming. :D
I'll change it and make a commit.